src/mgr/DaemonHealthMetricCollector.cc

```diff
@@ -37,15 +37,15 @@ class SlowOps final : public DaemonHealthMetricCollector {
     if (daemons.empty()) {
       return;
     }
-    static const char* fmt = "%1% slow ops, oldest one blocked for %2% sec";
-    check.summary = boost::str(boost::format(fmt) % value.n1 % value.n2);
+    static const char* fmt = "%1% slow ops, oldest one blocked for %2% sec, %3%";
     ostringstream ss;
     if (daemons.size() > 1) {
       ss << "daemons " << daemons << " have slow ops.";
     } else {
       ss << daemons.front() << " has slow ops";
     }
-    check.detail.push_back(ss.str());
+    check.summary = boost::str(boost::format(fmt) % value.n1 % value.n2 % ss.str());
```

Comment:
looks to me like this puts "NNN daemons have slow ops" in the summary?

Reply:
"log [WRN] : Health check update: 2 slow ops, oldest one blocked for 57 sec, daemons [osd.3,mon.a] have slow ops. (SLOW_OPS)"

That's what it does, so they get logged.